client/validators/profile.ts

```diff
@@ -38,6 +40,14 @@ export const formProfile = ({field, value, profile, errors, messages}) => {
     } else if (schema.required && (typeof fieldValue === 'number' ? !fieldValue : isEmpty(fieldValue))) {
         errors[field] = gettext('This field is required');
         messages.push(gettext('{{ name }} is a required field', {name: fieldLabel}));
+    } else if (schema.required && schema.multilingual) {
+        const errorMessage = gettext(`${fieldLabel} is a required field`);
+
+        if (field !== 'language' && diff?.languages?.length !== field_value?.length || field_value.some((obj
+        ) => obj.value === '')) {
+            errors[field] = gettext('This field is required');
```

Comment:
Do these errors appear under the field that is required? i.e. if the Canadian French version of `name` is not filled, is the error displayed underneath this field (might need to store the error under the `field` and `language` from the `translations` attribute entry